src/modules/recipe/lightDomHello/__tests__/lightDomHello.test.js

```diff
@@ -0,0 +1,33 @@
+import { createElement } from 'lwc';
+import LightDomHello from 'recipe/lightDomHello';
+
+describe('recipe-light-dom-hello', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('displays greeting in light DOM', () => {
+        const element = createElement('recipe-light-dom-hello', {
+            is: LightDomHello
+        });
+        document.body.appendChild(element);
+
+        // Verify displayed greeting
+        const divEl = element.querySelector('div');
```

Comment:
This isn't testing Light DOM since this div is directly in the component.
We need to query for a specific div in a child component with Light DOM.

I see that you did that so we should just keep `ligthDomQuery` and remove `lightDomHello`.

Reply:
@pozil Just the test, or the recipe altogether?